src/config.ts

```diff
@@ -1,20 +1,31 @@
-import path from "path";
-import yaml from "js-yaml";
+import * as core from "@actions/core";
 import * as github from "@actions/github";
+import * as Context from '@actions/github/lib/context';
+import yaml from "js-yaml";
+import path from "path";
 import { ConfigEntry } from "./ConfigEntry";
-
 const CONFIG_PATH = ".github";
 
-export async function getConfig(github: github.GitHub, fileName: string, { owner, repo }): Promise<ConfigEntry[]> {
+export async function getConfig(github: github.GitHub, fileName: string, context: Context.Context): Promise<ConfigEntry[]> {
+  console.log('getConfig context', context);
   try {
-    const response = await github.repos.getContents({
-      owner,
-      repo,
-      path: path.posix.join(CONFIG_PATH, fileName)
-    });
-
+    const configFile = {
+      owner: context.repo.owner,
+      repo: context.repo.repo,
+      path: path.posix.join(CONFIG_PATH, fileName),
+      ref: context.payload.pull_request!.head.sha,
+    };
+    core.debug(`Getting contents of ${JSON.stringify(configFile)}`);
+    const response = await github.repos.getContents(configFile);
+    if (Array.isArray(response.data)) {
+      throw new Error(`${fileName} is not a file.`);
+    }
+    if (response.data.content === undefined) {
+      throw new Error(`${fileName} is empty.`);
+    }
     return parseConfig(response.data.content);
   } catch (error) {
+    core.debug(`getConfig error: ${JSON.stringify(error)}`);
```

Comment:
```suggestion
    core.error(`getConfig error: ${JSON.stringify(error)}`);
```

Reply:
The reason I have this as a debug here is because when running jest, this line will be hit, so even when running `--silent` you end up seeing it. So as you can see in main.ts line 67 now, when this actually bubbles up in a real scenario, then it would get logged as a true error. But for testing we can test the `.toThrow` scenario without polluting our output.

Reply:
Sounds reasonable 👍🏻